index.html

```diff
@@ -1,26 +1,55 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Static Template</title>
-    <link rel="preconnect" href="https://fonts.gstatic.com" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;700;900&display=swap"
-      rel="stylesheet"
-    />
-    <link rel="preconnect" href="https://fonts.gstatic.com" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Indie+Flower&family=Roboto:wght@100;700;900&display=swap"
-      rel="stylesheet"
-    />
-  </head>
-  <body>
-    <h1 class="header" id="header1">Hola <b>#BootcampMex</b> 2021!</h1>
-    <h2 class="header" id="header2">Hola <b>#BootcampMex</b> 2021!</h1>
-    <h3 class="header" id="header3">Hola <b>#BootcampMex</b> 2021!</h1>
-    <h4 class="header" id="header4">Hola <b>#BootcampMex</b> 2021!</h1>
-  </body>
-</html>
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="./styles/master.css" rel="stylesheet" />
+  <title>Magazine</title>
+</head>
 
+<body>
+  <div class="magazine">
+    <div class="magazine__container"> 
+      <header class="conteiner__header">
+        <h1 class="header--title"> Traveling to Everest </h1>
+        <p class="header--description">Destination for climbers and adventure-seekers.</p>
+      </header>      
+      <section class="conteiner__about">
+        <ul class="about__block1">
```

Comment:
ahora tienes un uso excesivo de elementos ul 
los elementos heading and p no pueden ser children de UL

Reply:
puedes validar tu HTML en el sitio 

https://validator.w3.org/nu/#textarea

![image](https://user-images.githubusercontent.com/81988661/121737291-45e27c80-cabe-11eb-8f85-ebb8c7db9d91.png)